superset/sqllab/utils.py

```diff
@@ -56,3 +74,46 @@ def write_ipc_buffer(table: pa.Table) -> pa.Buffer:
         writer.write_table(table)
 
     return sink.getvalue()
+
+
+def bootstrap_sqllab_data(user_id: Union[int, None]) -> dict[str, Any]:
```

Comment:
@justinpark you can use `int | None` but you need to add `from __future__ import annotations` as the first import.
```suggestion
def bootstrap_sqllab_data(user_id: int | None) -> dict[str, Any]:
```